record-dot-preprocessor.cabal

```diff
@@ -69,12 +69,14 @@ test-suite record-dot-preprocessor-test
         base == 4.*,
         extra,
         record-hasfield,
-        filepath
+        filepath,
+        beam-core
     if impl(ghc >= 8.6)
         build-depends:
             record-dot-preprocessor
     other-modules:
         PluginExample
+        PluginHkdExample
```

Comment:
Why not add the plugin example to `PluginExample`? Why a specific module for this one?